src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -321,6 +324,7 @@ const wchar_t* mainWindowTitle = L"Endless Installer";
 
 #define ALL_FILES					L"*.*"
 
+#define CODING_INSTALLER_NAME    L"coding-installer.exe"
```

Comment:
Oh yeah, one more thing: if the brand is "Endless Code", shouldn't this be `code-installer.exe` or `endless-code-installer.exe`?

Also, at present you'll have "Endless OS" strings in the uninstaller.